lib/ansible/modules/windows/win_timezone.ps1

```diff
@@ -19,28 +19,32 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
-$params = Parse-Args $args;
-
-$result = New-Object psobject @{
-    win_timezone = New-Object psobject
+$result = @{
```

Comment:
Well, I didn't want to change the module in other ways then just the things I now changed.
Things like this we have to discuss with the Windows working group.